examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Example
+
+This directory contains example projects for the [JetBrains family of IDEs](https://www.jetbrains.com/products.html) that demonstrate how to work with the plugin and use it to deploy and develop on Kubernetes.
+
+*Note*: Before you can deploy and develop on Kubernetes with these projects, you need to make sure you have access to the image repository where the project image is about to be pushed. By default project is configured to use [Google Container Registry](https://cloud.google.com/container-registry/) `grc.io/gcp-dev-tools` repository which you probably don’t have access to. Once you have your own repository set up ([Google Container Registry](https://cloud.google.com/container-registry/), [DockerHub](https://hub.docker.com/), private repository, etc.), you can edit run targets and specify it as a *default image repository* in run target settings:
```

Comment:
Suggestion to simplify and merge both notes:

> Note: before deploying the application to Kubernetes using these samples, ensure that you update the image path to match your registry and image name. By default it is configured to use the [Google Container Registry](https://cloud.google.com/container-registry/). A simple way to do this without updating the image string in all locations is to set a _default image repository_ in the Kubernetes Run Configuration.

Reply:
Looks good, thanks! Updating.